realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -1707,16 +1713,17 @@ void BaseRealSenseNode::frame_callback(rs2::frame frame)
 
                 if (_enable_bright_region_removal)
                 {
-                    rs2::frameset fs = frame.as<rs2::frameset>();
-                    auto ir = fs.as<rs2::frameset>().get_infrared_frame(1);
-                    if (ir)
+                    static int count_total = 0, count_skipped = 0; 
+                    count_total++;
```

Comment:
modern compilers may not care about this so much, but as good coding practice its better to use prefix increment than postfix (technically for efficiency)